src/app/GitUI/CommandsDialogs/FormAddFiles.cs

```diff
@@ -15,13 +15,13 @@ public FormAddFiles(IGitUICommands commands, string? addFile = null)
 
         private void ShowFilesClick(object sender, EventArgs e)
         {
-            string arguments = string.Format("add --dry-run{0} \"{1}\"", force.Checked ? " -f" : "", Filter.Text);
+            string arguments = string.Format("add --dry-run{0} {1}", force.Checked ? " -f" : "", Filter.Text);
```

Comment:
What's the value of `Filter.Text`? Doesn't it need to be quoted? That is `Filter.Text.Quote()`?

Reply:
> What's the value of Filter.Text? Doesn't it need to be quoted? That is Filter.Text.Quote()?

Command line arguments have already been quoted (refer to the screenshot in the PR description). Other callsites do not fill the textbox.
So if the user removes quotes from file names with spaces, it is their fault.

Reply:
Makes sense